pkg/generate/app/env.go

```diff
@@ -24,6 +30,26 @@ func ParseEnvironment(vals ...string) Environment {
 	return env
 }
 
+func ParseEnvironment(vals ...string) (Environment, []string, []error) {
```

Comment:
Added `IsValidArgumentEnvironment` check to `ParseEnvironment` as well as `LoadEnvironmentFile` because godotenv library allows such keys.